skywatch/info.json

```diff
@@ -46,51 +54,49 @@
     "layout_aliases": {
```

Comment:
While at it, should we remove this block as well, since with this PR it supports community layouts and all definitions are modified?

Reply:
removed as per commit [fbc9a04](https://github.com/ozkan/Skywatch-QMK/pull/1/commits/fbd9a045900f1dea85aa8e5afd3364c3aeb4e2f9) 🙂